### PR TITLE
feat(git): purge jsdelivr cdn cache workflow

### PR DESCRIPTION
**Pull request by bigboydiamonds**
## Summary by CodeRabbit

- **Chores**
  - Introduced a new workflow to purge CDN caches (jsDelivr) on specific push events, improving cache management and ensuring up-to-date content delivery.

### CODE REVIEW

File: .github/workflows/purge.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    paths:
+      - 'packages/synapse-interface/public/pauses/v1/**'
+
+jobs:
+  purge-cache:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```
Comment (trajan0x) [inline .github/workflows/purge.yml:14]: not needed

File: .github/workflows/purge.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+      - name: Purge CDN Caches
+        uses: gacts/purge-jsdelivr-cache@v1
+        with:
+          url: |
+            https://cdn.jsdelivr.net/gh/synapsecns/sanguine@master/packages/synapse-interface/public/pauses/v1/paused-chains.json
+            https://cdn.jsdelivr.net/gh/synapsecns/sanguine@master/packages/synapse-interface/public/pauses/v1/paused-bridge-modules.json
+            https://cdn.jsdelivr.net/gh/synapsecns/sanguine@test/maintenance/packages/synapse-interface/public/pauses/v1/paused-chains.json
```
Comment (trajan0x) [inline .github/workflows/purge.yml:22]: should nuke these before merge

**Review by trajan0x — CHANGES_REQUESTED** [review]

File: .github/workflows/purge.yml (hunk 1 above)
Comment (bigboydiamonds) [reply .github/workflows/purge.yml]: removed in https://github.com/synapsecns/sanguine/pull/2857/commits/ee521e1c39219ac5b072e6a31ad6d7f58a3ec456

File: .github/workflows/purge.yml (hunk 2 above)
Comment (bigboydiamonds) [reply .github/workflows/purge.yml]: removed test cdn links: https://github.com/synapsecns/sanguine/pull/2857/commits/ee521e1c39219ac5b072e6a31ad6d7f58a3ec456

**Pull request closed without merging**